src/transformers/utils/import_utils.py

```diff
@@ -169,6 +169,176 @@ def _is_package_available(pkg_name: str, return_version: bool = False) -> Union[
         logger.info("Disabling Tensorflow because USE_TORCH is set")
 
 
+if USE_JAX in ENV_VARS_TRUE_AND_AUTO_VALUES:
```

Comment:
The code diff looks pretty mega here - should just be these two new checks no?
```python
_essentia_available = importlib.util.find_spec("essentia") is not None
try:
    _essentia_version = importlib_metadata.version("essentia")
    logger.debug(f"Successfully imported essentia version {_essentia_version}")
except importlib_metadata.PackageNotFoundError:
    _essentia_version = False

_pretty_midi_available = importlib.util.find_spec("pretty_midi") is not None
try:
    _pretty_midi_version = importlib_metadata.version("pretty_midi")
    logger.debug(f"Successfully imported essentia version {_pretty_midi_version}")
except importlib_metadata.PackageNotFoundError:
    _pretty_midi_available = False
```

Reply:
I am so sorry about this, I surely messed up while resolving conflicts!